src/drivers/imu/bosch/bmi088/BMI088_Gyroscope.cpp

```diff
@@ -427,6 +427,8 @@ bool BMI088_Gyroscope::FIFORead(const hrt_abstime &timestamp_sample, uint8_t sam
 	gyro.samples = samples;
 	gyro.dt = FIFO_SAMPLE_DT;
 
+	int index = 0;
```

Comment:
Seems pretty unlikely, but probably best to only call `_px4_gyro.updateFIFO(gyro);` (line 452 below) if index > 0.

Reply:
@dagar san.
Thanks for the review.
I have fixed it.
I also fixed the I2C driver.